$NetBSD: patch-mc,v 1.3 2009/06/30 20:43:37 tnn2 Exp $

We manage the optimizer flags ourselves, I guess?
XXX but we still pass --enable-optimize=-O2 to configure?

--- security/coreconf/command.mk.orig	2009-06-29 18:15:11.000000000 +0200
+++ security/coreconf/command.mk
@@ -45,7 +45,7 @@ ASFLAGS      += $(CFLAGS)
 CCF           = $(CC) $(CFLAGS)
 LINK_DLL      = $(LINK) $(OS_DLLFLAGS) $(DLLFLAGS)
 LINK_EXE      = $(LINK) $(OS_LFLAGS) $(LFLAGS)
-CFLAGS        = $(OPTIMIZER) $(OS_CFLAGS) $(XP_DEFINE) $(DEFINES) $(INCLUDES) \
+CFLAGS        = $(OS_CFLAGS) $(XP_DEFINE) $(DEFINES) $(INCLUDES) \
 		$(XCFLAGS)
 PERL          = perl
 RANLIB        = echo
